### PR TITLE
linux/a*: fix German translation

### PR DESCRIPTION
**Pull request by marcelbrode**
Corrected mostly compound words

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/apt-add-repository.md [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
 `apt-add-repository --update {{repository}}`
 
-- Aktiviere Source Pakete:
+- Aktiviere Source-Pakete:
```
Comment (marchersimon) [inline pages.de/linux/apt-add-repository.md:18]: Finally someone who understands the struggle with the missing dashes 😅

File: pages.de/linux/apt-file.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # apt-file
 
-> Suche nach Dateien in apt Paketen, auch in den nicht installierten.
+> Suche nach Datein in apt-Paketen, auch in den nicht-installierten.
```
Comment (marchersimon) [inline pages.de/linux/apt-file.md:3]: I'm pretty sure it's `Dateien`, not `Datein`.
```suggestion
> Suche nach Dateien in apt-Paketen, auch in den nicht-installierten.
```

File: pages.de/linux/asciiart.md [hunk 3]
```diff
@@ -23,6 +23,6 @@
 
 `asciiart --format {{text|html}} {{pfad/zum/bild.jpg}}`
 
-- Invertiere die Buchstaben Map:
+- Invertiere die Char Map:
```
Comment (marchersimon) [inline pages.de/linux/asciiart.md:26]: How about this?
```suggestion
- Invertiere die Zeichentabelle:
```

File: pages.de/linux/apt-file.md (hunk 2 above)
Comment (marcelbrode) [reply pages.de/linux/apt-file.md:3]: Oh well, that wasn't on purpose.

File: pages.de/linux/asciiart.md (hunk 3 above)
Comment (marcelbrode) [reply pages.de/linux/asciiart.md:26]: Waybetter 🤦 my bad

**Review by marchersimon — APPROVED** [review]
Thanks @marcelbrode

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**